### PR TITLE
Add aventus

### PR DESCRIPTION
**Pull request by dr-ngubo**
##### Twitter Link:
https://twitter.com/aventusnetwork

##### List of audit links if any:
https://drive.google.com/file/d/1zsQ7fnfxSM8HWZRs6eg16N49YZ3n4j9W/view?usp=sharing

##### Website Link:
https://www.aventus.io/

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
https://drive.google.com/file/d/1MT5yNJdHRB5d_fm1DoJKqkIAdEuRSJjH/view?usp=sharing 

##### Current TVL:
4.47 M

##### Chain:
Aventus

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
aventus

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
1948

##### Short Description (to be shown on DefiLlama):
The customisable layer-2 blockchain network that lets you build on Ethereum at scale to process transactions at 100x the speed and 1% of the cost.

##### Token address and ticker if any:
0x0d88ed6e74bbfd96b831231638b66c05571e824f on Ethereum
AVT

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Cross Chain

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
No

##### forkedFrom (Does your project originate from another project):
No

##### methodology (what is being counted as tvl, how is tvl being calculated):
Staking
Total AVT on the Aventus network.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/aventus exports TVL: 
        
 
 
    aventus                   4.39 M
    aventus-staking           88.00 k
    staking                   88.00 k
    
    total                    4.39 M

File: projects/aventus/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      await axios.get(
+        "https://api.coingecko.com/api/v3/simple/price?ids=aventus&vs_currencies=usd&include_market_cap=true&include_24hr_vol=true&include_24hr_change=true"
+      )
+  );
+
+  // Initialise Aventus Network Gateway API and Fetch data
+  await API.init();
+
+  // Total amount of AVT on the Aventus Network
+  var totalAvt = await API.query.getTotalAvt();
```
[the remark this answers is not in the document]
Comment (dr-ngubo) [reply projects/aventus/index.js:25]: This is counting all AVT moved to the network from Ethereum by users to pay for their transactions.
The smart contract for the AVT token sits on Ethereum.

File: projects/aventus/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  var tvl = new BigNumber(totalAvt).div(10 ** 18).toFixed(2);
+  return tvl * price_feed.data.aventus.usd;
+}
+
+async function staking() {
+  // Initialise Aventus Network Gateway API and Fetch data
+  await API.init();
+
+  // Total amount of AVT Staked on the Aventus Network
+  var stakingStats = await API.query.getStakingStats();
```
[the remark this answers is not in the document]
Comment (dr-ngubo) [reply projects/aventus/index.js:35]: This includes both the staking deposit by each validator and the independent staking by the community.

File: projects/aventus/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (waynebruce0x) [reply projects/aventus/index.js:25]: For the tokens to be counted they need to be locked in a defi protocol as well as being on the chain

File: projects/aventus/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (waynebruce0x) [reply projects/aventus/index.js:35]: https://docs.llama.fi/list-your-project/what-to-include-as-tvl
https://docs.llama.fi/list-your-project/readme

**Pull request closed without merging**